bin/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "exocom-publish",
```

Comment:
Make the name something like `exocom publish script - don't deploy!` to make sure nobody deploys it.